### DISCUSSION TITLE
Solidos landing page: on https://solidos.solidcommunity.net

### DISCUSSION THREAD

**Discussion by bourgeoa** [solid/solidos#75] [Archive] [answered]
@theRealImy thanks for the new page.
1. The page is static I suppose this is the intention. but do not give access to root of the folder.
Actually solidos can only give access to the podWebId : https://solidos.solidcommunity.net/profile/card#me. All other must now the links
- Access to root should be allowed to all authorized members (agent, group members, ....).

- Need to make sure the page works in the new authorization upgrade

- The old page allowed this by displaying at the bottom the content of the default mashlib index.html with a login and a folder-pane access.

   Here is the old page code : 
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Solid Data Browser</title>
    <script>
      document.addEventListener('DOMContentLoaded', function() {
        panes.runDataBrowser()
      })
    </script>
  <link href="/mash.css" rel="stylesheet"></head>
  <body class="db-layout">
    <h2>Welcome to the SolidOS Pod</h2>
    <p> Useful links:</p>
    <ul>
      <li><a href="https://solidos.solidcommunity.net/Team/2021/schedule/solidos-schedule.html">🎯 Team meeting link & schedule</a> </li>
      <li><a href="https://solidos.solidcommunity.net/Team/SolidOs%20team%20chat/index.ttl#this">🎧 Team chat</a></li>
      <li><a href="https://github.com/solid/solidos">💻 SolidOS on GitHub</a>
      <li><a href="https://solidos.solidcommunity.net/Team/docs/solidos.html">🌟 SolidOS north star</a></li>
    </ul>
    <!-- solid-panes' OutlineManager injects into this element -->
    <header class="db-layout__header header" id="PageHeader"></header>
    <div class="TabulatorOutline db-layout__content" id="DummyUUID" role="main">
        <table id="outline"></table>
        <div id="GlobalDashboard"></div>
    </div>
    <footer class="db-layout__footer" id="PageFooter"></footer>
  <script type="text/javascript" src="/mashlib.min.js"></script></body>
</html>
```

2. I feel the banner header is much too big and character size used on the rest of page also.

3. the page correctly display on a phone

**Comment by angelo-v** [comment]
Not sure if the root folder access is really needed. If it is easy to integrate it, fine, other option could be to move relevant files to the public folder or other subfolders

**Comment by bourgeoa** [comment]
I think there are 2 reasons to inclure access to root folder : 
- a proof concept. That's why I liked the implementation in the previous one.
- the fact that there are 2 shared folders : Contacts and Team, apart the usual private folders (private,  inbox, settings).
  - Moving Contacts implies to redo the group access links. With the risk (in theory) that some could be used outside the pod and you cannot know.
  - Moving Team to public can be done using external app like https://jeff-zucker.github.io/solid-ide/ or https://otto-aa.github.io/solid-filemanager/. I suppose that making a link to Team in the landing page is a better idea.

**Comment by jeff-zucker** [comment]
Great work @theRealImy, thanks.  I agree that the header is larger than I'd like - I'd prefer the logo and banner on a single line to not take up that amount of vertical real estate.  The font size on the page works for me.   My only question is what does "North Star" refer to?  Is that another way of saying "Vision Statement"?  Might be more inclusive to use a more descriptive name for the page.  Other than that, looks great.

**Comment by timea-solid** [comment]
Thank you for all the great feedback! I will head over and implement it.
I gotta admit I think how the SolidOS is structured (public, Team not in public, 2020 but 2021 also...) is not intuitive. Even with the previous index.html I do not know how to find the Team page or the docs. I am glad this is also reflected now in the discussion. 
My intention: have the solidOS index.html act like a webpage for the project.  The webpage is targetting:
- people wanting to read about SolidOS vision and current progress
- SolidOS team
- maybe also people who want to see SolidOS in action. 

Question to @bourgeoa: root folder access - what do you exactly mean with this? Because I do not think I had that before either. 
Access to index.html is done through Dokieli now.

**Comment by bourgeoa** [comment]
root folder access should have worked by clicking on the folder-pane.
I forgive that I must make a PR to make it appear when url ends with index.html

**Comment by timea-solid** [comment]
@bourgeoa  ok. That would be cool if I can see it in the future! Question: should this be public or private? Should everyone see it/access it/be able to edit it?

**Comment by timea-solid** [comment, marked as answer]
Most of the feedback was implemented.  And thank you again for helping me focus what this Pod can and should become. 

I intent to keep the Pod up to date and improve it constantly as part of the idea of 'eat your own dog food'. So stay tuned ;) 

The goal here is to: use the Pod as an immediate example of the implementation and usage of SolidOS on a Pod and make it a central point for the team too. This is the reason why I changed the welcome text on the front page. 

Regarding moving the Team pages under public. -> I will postpone this to when we can delete resources (upon next release of SolidOS) because I am still thinking of the structure of the folders. I do not want to loose anything nor do a bulk move just yet.
